### PR TITLE
Build: Upgrades to golang 1.12.6

### LINKED ISSUE #17533: Backend: Upgrade to golang 1.12.6
https://groups.google.com/forum/#!msg/golang-announce/dNU0sAdX65I/3psx7_OqAQAJ

more info about how TBD

### PR DESCRIPTION
**Pull request by kylebrandt**
fixes #17533

### CODE REVIEW

File: scripts/build/ci-build/Dockerfile [hunk 1]
```diff
@@ -35,6 +35,8 @@ RUN apt-get update   && \
         curl            \
         git             \
         make            \
+        cmake           \
```
Comment (bergquist) [inline scripts/build/ci-build/Dockerfile]: Just to avoid errors?

Comment (kylebrandt) [reply scripts/build/ci-build/Dockerfile]: Required to build

Comment (bergquist) [reply scripts/build/ci-build/Dockerfile]: So weird that it worked for me first time. I guess some of my local containers was cached.

**Review by bergquist — APPROVED** [review]
LGTM

**Pull request merged**